test/git_blame_info_test.rb

```diff
@@ -33,6 +33,29 @@ def test_parse
       ], GitBlameInfo.parse(porcelain_output))
   end
 
+  def test_parse_for_none_utf_8
+    string_with_invalid_encoding = <<~DATA
+      144cae9deb69339061292e9db8541701197c640d 1 1 1
+      author Yutaka Kamei
+      author-mail <yutaka@sider.review>
+      author-time 1583907807
+      author-tz +0900
+      committer Yutaka Kamei
+      committer-mail <yutaka@sider.review>
+      committer-time 1583907807
+      committer-tz +0900
+      summary 2
+      previous af89ec8f2a0c54c0758e64046ac7f40bcaa7d5ac abc.txt
+      filename abc.txt
+      \t\x8C\xE1\x94y\x82͔L\x82ł\xA0\x82邩\x82\xE0\x82\xB5\x82\xEA\x82Ȃ\xA2\r\n
+    DATA
+    assert_raises(ArgumentError) { GitBlameInfo.parse(string_with_invalid_encoding) }
```

Comment:
[suggestion] ArgumentError is generic, so what about testing the message to distinguish the expected error?

For example:

```suggestion
    error = assert_raises(ArgumentError) { GitBlameInfo.parse(string_with_invalid_encoding) }
    assert_equal "invalid byte sequence in UTF-8", error.message
```

Reply:
Nice. Thank you!